src/plugins/discover/public/application/angular/doc_table/create_doc_table_embeddable.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { render } from 'react-dom';
+import React, { useRef, useEffect } from 'react';
+import { I18nProvider } from '@kbn/i18n/react';
+import { IScope } from 'angular';
+import { getServices } from '../../../kibana_services';
+import { DocTableLegacyProps, injectAngularElement } from './create_doc_table_react';
+
+type AngularEmbeddableScope = IScope & { renderProps?: DocTableEmbeddableProps };
+
+export interface DocTableEmbeddableProps extends Partial<DocTableLegacyProps> {
+  refs: HTMLElement;
+}
+
+function getRenderFn(domNode: Element, props: DocTableEmbeddableProps) {
+  const directive = {
```

Comment:
:art: Since this object has only one property, and doesn't seem to be used anywhere except for accessing that single property, maybe we could just make this a `const template` instead.

Reply:
For consistency with other angular directives and overall readability, I'd like to keep it as is